app.py

```diff
@@ -143,9 +150,211 @@ def get_accounts():
+        req_currency = instructed_amount["currency"]
+        with open("./res/balances.json") as f:
+            balances = json.load(f)
+            if req_iban not in balances.keys():
+                return err_resp
+            cur_amount = balances[req_iban][req_currency]
+        if req_amount > cur_amount:
+            headers["X-Request-ID"] = current_req_id
+            return Response(
+                json.dumps(
+                    {
+                        "fundsAvailable": "false"
+                    }),
+                status=200,
+                headers=headers
+            )
+        else:
+            headers["X-Request-ID"] = current_req_id
+            return Response(
+                json.dumps(
+                    {
+                        "fundsAvailable": "true"
+                    }),
+                status=200,
+                headers=headers
+            )
+    return err_resp
+
+
+@app.route(api_path + "/v1/payments/sepa-credit-transfers", methods=["POST"])
+@app.route(api_path + "/v1/payments/sepa-credit-transfers/", methods=["POST"])
+def transfer_sepa_credit():
+    header = request.headers
+    if header.get("X-Request-ID") == current_req_id and header.get("Consent-ID") == consent \
+            and header.get("TPP-Redirect-URI") == "site.com" and header.get("Content-Type") == "application/json":
+        tpp_redirect_uri = header.get("TPP-Redirect-URI")
+        instructed_amount = json.loads(request.data).get("instructedAmount")
+        req_amount = instructed_amount["amount"]
+        req_currency = instructed_amount["currency"]
+        debtor_account = json.loads(request.data).get("debtorAccount")
+        debtor_iban = debtor_account["iban"]
+        creditor_name = json.loads(request.data).get("creditorName")
+        creditor_account = json.loads(request.data).get("creditorAccount")
+        creditor_iban = creditor_account["iban"]
+        with open("./res/balances.json", "r+") as f1:
+            balances = json.load(f1)
+            if debtor_iban not in balances.keys() or creditor_iban not in balances.keys() \
+                    or req_currency not in balances[debtor_iban]:
+                return err_resp
+            global payment_id
+            payment_id = payment_id + 1
+            cur_amount_debtor = balances[debtor_iban][req_currency]
+            if req_amount > cur_amount_debtor:
+                headers["X-Request-ID"] = current_req_id
+                headers["TPP-Redirect-URI"] = tpp_redirect_uri
+                headers["ASPSP-SCA-Approach"] = "REDIRECT"
+                temp_payment_id = payment_id - 1
+                return Response(
+                    json.dumps(
+                        {
+                            "paymentId": temp_payment_id,
+                            "_links": {
+                                "scaRedirect": tpp_redirect_uri,
+                                "self": "/v1/payments/" + str(temp_payment_id),
```

Comment:
is this resource will now have a file or an entry with information regarding this payment transfer?